source/service_test.go

```diff
@@ -3600,11 +3603,32 @@ func TestExternalServices(t *testing.T) {
 				hostnameAnnotationKey: "service.example.org",
 			},
 			"remote.example.com",
+			[]string{},
 			[]*endpoint.Endpoint{
 				{DNSName: "service.example.org", Targets: endpoint.Targets{"remote.example.com"}, RecordType: endpoint.RecordTypeCNAME},
 			},
 			false,
 		},
+		{
+			"annotated ExternalName service with externalIPs returns a single endpoint with multiple targets",
+			"",
+			"testing",
+			"foo",
+			v1.ServiceTypeExternalName,
+			"",
+			"",
+			false,
+			map[string]string{"component": "foo"},
+			map[string]string{
+				hostnameAnnotationKey: "service.example.org",
+			},
+			"service.example.org",
+			[]string{"10.2.3.4", "11.2.3.4"},
```

Comment:
what do we do with a mix of ipv4 (A) and ipv6 (AAAA)?
I guess this should be also fine, but better test it as expected.

Reply:
Fair point, testcase added

Reply:
thanks